src/crypto/OperationalKeystore.h

```diff
@@ -168,6 +168,26 @@ class OperationalKeystore
      */
     virtual CHIP_ERROR SignWithOpKeypair(FabricIndex fabricIndex, const ByteSpan & message,
                                          Crypto::P256ECDSASignature & outSignature) const = 0;
+
+    /**
+     * @brief Create an ephemeral keypair for use in session establishment.
+     *
+     * WARNING: There may be more than one of these in use concurrently, so implementations have to
+     *          be careful to avoid assuming a single instance.
+     *
+     * WARNING: The return value MUST be released by `ReleaseEphemeralKeypair`. This is because
+     *          Matter CHIPMem.h does not properly support UniquePtr in a way that would
+     *          safely allow classes derived from Crypto::P256Keypair to be released properly.
+     *
+     * @return a pointer to a dynamically P256Keypair (or derived class thereof), which may evaluate to nullptr
+     *         if running out of memory.
+     */
+    virtual Crypto::P256Keypair * AllocateEphemeralKeypair() = 0;
```

Comment:
This should document that the caller is expected to call `Initialize()`, right?

That said, why is the caller expected to do that?  Can't the implementation here handle any needed initialization?

Reply:
Initialize is not needed when deserializing. I kept the existing convention of not assuming Initialize, which is the caller's role to do. Initialize costs an EC key pair generation, which doesn't have zero cost.

Reply:
But are there any cases where someone does `AllocateEphemeralKeypair` and then deserializes into it, instead of doing an `Initialize`?

Reply:
No, I don't think so. The only usage is for one to be initialized. I could add that change as a follow-up, or I will do it if CI fails and I need to repush

Reply:
Done.